app/src/main/java/com/google/android/ground/ui/map/gms/GoogleMapsFragment.kt

```diff
@@ -226,6 +226,8 @@ class GoogleMapsFragment : SupportMapFragment(), MapFragment {
     val clickedPolygons = featureManager.getIntersectingPolygons(latLng)
     if (clickedPolygons.isNotEmpty()) {
       viewLifecycleOwner.lifecycleScope.launch { featureClicks.emit(clickedPolygons) }
+    } else {
+      viewLifecycleOwner.lifecycleScope.launch { featureClicks.emit(emptySet()) }
     }
```

Comment:
Can this be simplified by merging the two?

Reply:
Right! Done.